lib/logging/include/hse/logging/logging.h

```diff
@@ -91,7 +91,7 @@ void
 logging_fini(void) HSE_COLD;
 
 merr_t
-logging_init(const struct logging_params *params) HSE_COLD;
+logging_init(const struct logging_params *params, merr_stringify ctx_stringify) HSE_COLD;
```

Comment:
I like this better, "ctx_stringify".

Reply:
Cool. Will adjust the other location accordingly